openedx/features/qverse_features/registration/utils.py

```diff
@@ -0,0 +1,17 @@
+"""
+Qverse registration application utils functions.
+"""
+from django.contrib.auth.models import User
+
+
+def get_user_by_regno_or_username(identifier):
```

Comment:
```suggestion
def get_user_by_username(identifier):
```
????

Reply:
I think both are fine.

Reply:
`registration_number` is an actual field in the model but here, only `username` is being used for the query.
This name can cause misunderstanding.

Reply:
done.